<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimulateMatch</class>
 <widget class="QWidget" name="SimulateMatch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ourTeamLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="opponentTeamLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ourGoals">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="opponentGoals">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>37</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string>This is the final score of the match based on the starting players statistics.</string>
   </property>
   <property name="text">
    <string>i</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
